generators/Exercises/Acronym.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Generators.Exercises
+{
+    public class Acronym : EqualityExercise
+    {
+        public Acronym() : base("acronym")
```

Comment:
Another potential simplification would be to generate the exercise name from the Class name using reflection. The rules are pretty simple:
1. Acronym -> acronym
2. NthPrime -> nth-prime

You could argue it actually makes the code more obscure but it's some food for thought :)